include/mgmt/mgmt_fmfu.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file mgmt_fmfu.h
+ * @defgroup mgmt_fmfu Management Full Modem Firmware Upgrade 
+ * @{
+ * @brief Full Modem Firmware Update(FMFU).
```

Comment:
State already here that it is SMP based, otherwise this brief can be confused with OTA